caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -1066,7 +1068,13 @@ def _execute_train_loop(
             num_training_steps=(len(train_dataloader) * num_epochs),
         )
         # Configure accelerator for gradient accumulation
-        accelerator = Accelerator(gradient_accumulation_steps=accumulate_steps)
+        accelerator_args = {
+            "gradient_accumulation_steps": accumulate_steps,
+            "device_placement": True,
+        }
+
+        accelerator = Accelerator(**accelerator_args)
```

Comment:
why build a separate dict here?

Reply:
I was playing with some optional parameter regarding `cpu=True`.. But that didn't work well, so removed that.. So this is kinda left over from that.. Will switch it back to direct arguments instead of separate dict.